src/fs/filesystem.ts

```diff
@@ -172,6 +177,7 @@ export class FileSystem {
     return cid
   }
 
+  // Legit don't know what this means: synchronize vs synchronous
```

Comment:
synchronize

yeah I didn't realize the naming overlap here, I'm happy to change that up